packages/web3-eth-accounts/src/index.js

```diff
@@ -162,13 +162,11 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
         }
 
         try {
-            tx = helpers.formatters.inputCallFormatter(tx);
-
-            var transaction = tx;
-            transaction.to = tx.to || '0x';
-            transaction.data = tx.data || '0x';
-            transaction.value = tx.value || '0x';
-            transaction.chainId = utils.numberToHex(tx.chainId);
+            var transaction = helpers.formatters.inputCallFormatter(JSON.parse(JSON.stringify(tx)));
```

Comment:
Could you use the [clone](https://underscorejs.org/#clone) method of the anyways existing ``underscore`` package and update the CHANGELOG.md file in the root folder of the 1.x branch?

Reply:
@nivida: Done.